[TECH] Mettre à jour ember-page-title sur Pix APP (PIX-1738)

sbedeau opened the pull request:
## :unicorn: Problème
Le plugin `ember-page-title` n'est pas à jour et génère un warning à la transpilation depuis Ember 3.20 #2179 

```
DEPRECATION: The use of the private `{{-in-element}}` is deprecated, please refactor to the public `{{in-element}}`. ('ember-cli-head/templates/components/head-layout.hbs' @ L1:C0)  [deprecation id: glimmer.private-in-element]
        at logDeprecationStackTrace (/home/circleci/pix/mon-pix/node_modules/ember-source/dist/ember-template-compiler.js:1930:21)
        at HANDLERS.<computed> (/home/circleci/pix/mon-pix/node_modules/ember-source/dist/ember-template-compiler.js:2027:9)
        at raiseOnDeprecation (/home/circleci/pix/mon-pix/node_modules/ember-source/dist/ember-template-compiler.js:1957:9)
        at HANDLERS.<computed> (/home/circleci/pix/mon-pix/node_modules/ember-source/dist/ember-template-compiler.js:2027:9)
        at invoke (/home/circleci/pix/mon-pix/node_modules/ember-source/dist/ember-template-compiler.js:2039:9)
        at deprecate (/home/circleci/pix/mon-pix/node_modules/ember-source/dist/ember-template-compiler.js:1995:28)
        at BlockStatement (/home/circleci/pix/mon-pix/node_modules/ember-source/dist/ember-template-compiler.js:10730:59)
        at visitNode (/home/circleci/pix/mon-pix/node_modules/ember-source/dist/ember-template-compiler.js:7742:16)
        at visitArray (/home/circleci/pix/mon-pix/node_modules/ember-source/dist/ember-template-compiler.js:7833:20)
```

## :robot: Solution
- Mettre à jour `ember-page-title` en [version ^6.0.0](https://github.com/ember-cli/ember-page-title/releases/tag/v6.0.0) 
- Se mettre en conformité avec les breaking changes proposés dont [l'indépendance avec `ember-cli-head`](https://github.com/ember-cli/ember-page-title/pull/168)

## :rainbow: Remarques
Suite à la prise d’indépendance avec `ember-cli-head`, on obtient l'erreur suivante au moment d'accéder à l'app et dans les tests : 
```
Uncaught (in promise) Error: Assertion Failed: [ember-page-title]: Multiple title elements found. Check for other addons like ember-cli-head updating <title> as well.
```

~~On se retrouve donc à devoir choisir comment faire nos titres de pages :~~
~~• soit on est obligé de supprimer le `<title>` dans `index.html` et ajouter `Pix |` dans tous nos titres à la main~~
~~• soit on conserve notre approche (composition de titres) actuelle mais il faut se séparer d’`ember-page-title` au profit (intégral ?) d’`ember-cli-head`~~
~~• soit il y a une option dans `ember-page-title` que je n’ai pas encore trouvé~~

💡 L'[upgrading-guide](https://github.com/ember-cli/ember-page-title#upgrading-notes-for-5x-to-6x) nous indique de supprimer `<HeadLayout />` dans `application.hbs`
_Note pour plus tard : c'est appréciable de trouver l'upgrading guide directement dans la note de version majeure..._

❓ Nous utilisons `ember-cli-head` dans nos paquets. Est-ce qu'on en a encore besoin ? 
A priori oui pour injecter des données dans le `head` via le [`Head Data Service`](https://github.com/ronco/ember-cli-head#head-data-service).

Les tests liés aux titres en font largement usage : 
```
Error: Assertion Failed: Attempting to inject an unknown injection: 'service:headData'
```

## :100: Pour tester
Vérifier le bon affichage des titres sur l'application

Commit:
chore: bump ember-page-title

pix-service commented:
I'm deploying this PR to these urls:

- App: https://app-pr2233.review.pix.fr
- Orga: https://orga-pr2233.review.pix.fr
- Certif: https://certif-pr2233.review.pix.fr
- Admin: https://admin-pr2233.review.pix.fr
- API: https://api-pr2233.review.pix.fr/api/

 Please check it out!

Commit:
chore: remove no more necessary tag

Cf. [upgrading guide](https://github.com/ember-cli/ember-page-title#upgrading-notes-for-5x-to-6x)
for details.

Commit:
chore: follow stylelint recommandation

jbuget reviewed (approved):
Beau boulot : 
- découpage des commits
- titre et description de la PR
- contenu
👏
